From a7c5d6d6e7e1e23a4e9862906ce9c3375cc7d96b Mon Sep 17 00:00:00 2001
From: xoviat <xoviat@users.noreply.github.com>
Date: Fri, 19 Jan 2018 17:36:31 -0600
Subject: [PATCH] BLD: Remove hack for windows

---
 scikits/umfpack/setup.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/scikits/umfpack/setup.py b/scikits/umfpack/setup.py
index 3088aaa..7218284 100644
--- a/scikits/umfpack/setup.py
+++ b/scikits/umfpack/setup.py
@@ -29,13 +29,6 @@ def umfpack_i(ext, build_dir):
     dict_append(build_info, **umf_info)
     dict_append(build_info, **blas_info)
     
-    # On windows, vcpkg names the library 'libumfpack'
-    # rather than 'umfpack' and this change is not yet
-    # in numpy; therefore, we fix the problem here
-    if sys.platform == 'win32':
-        build_info['libraries'].remove('umfpack')
-        build_info['libraries'].append('libumfpack')
-
     config.add_extension('__umfpack',
                          sources=[umfpack_i],
                          depends=['umfpack.i'],
